mitiq/tests/test_folding_cirq.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import random
+import cirq
+from mitiq.folding_mitiq import local_folding, unitary_folding
+
+def random_circuit(depth):
+    """Returns a single-qubit random circuit based on Pauli gates."""
+    # defines qubit
+    q = cirq.GridQubit(0, 0)
+    circuit = cirq.Circuit()
+    for _ in range(depth):
+        circuit += random.choice([cirq.X(q), cirq.Y(q), cirq.Z(q)])
+    return circuit
+
+strech_vals = [1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.5, 4.5, 5.0]
+depth = 100
```

Comment:
A good styling principle is that for global variables at the module level one should label them in all caps to distinguish their scope and prevent weird artifacts of global state.
```suggestion
STRETCH_VALS = [1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.5, 4.5, 5.0]
DEPTH = 100
```